### PR TITLE
Wombex strategy

### PR DESCRIPTION
**Pull request by AlehNat**

### CODE REVIEW

File: contracts/strategies/UniversalLendStrategy.sol [hunk 1]
```diff
@@ -198,27 +198,29 @@ abstract contract UniversalLendStrategy is ProxyStrategyBase {
...
+          if (remaining != 0) {
+            localBalance += remaining;
+          }
         }
       }
+
     }
     IBookkeeper(c.bookkeeper()).registerStrategyEarned(0);
-    return _getActualPoolBalance();
+    return poolBalance;
```
Comment (belbix) [inline contracts/strategies/UniversalLendStrategy.sol]: why?

File: contracts/strategies/radiant/Radiant2StrategyBase.sol [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  /// @dev Deposit to pool and increase local balance
+  function _simpleDepositToPool(uint amount) internal override {
+    address u = _underlying();
+    _approveIfNeeds(u, amount, address(AAVE_LENDING_POOL));
+    AAVE_LENDING_POOL.deposit(u, amount, address(this), 0);
+  }
+
+  /// @dev Perform only withdraw action, without changing local balance
+  function _withdrawFromPoolWithoutChangeLocalBalance(uint amount, uint poolBalance) internal override returns (bool withdrewAll, uint withdrawnAmount) {
+    uint underlyingBalanceBefore = IERC20(_underlying()).balanceOf(address(this));
```
Comment (belbix) [inline contracts/strategies/radiant/Radiant2StrategyBase.sol]: reuse underlying in local variable

File: contracts/strategies/wombex/WombexStrategyBase.sol [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    uint minLiquidity = amount * (PRICE_IMPACT_PRECISION - PRICE_IMPACT) / PRICE_IMPACT_PRECISION;
+    POOL_DEPOSITOR.deposit(address(lpToken), amount, minLiquidity, block.timestamp + 1, true);
+  }
+
+  /// @dev Perform only withdraw action, without changing local balance
+  function _withdrawFromPoolWithoutChangeLocalBalance(uint amount, uint poolBalance) internal override returns (bool withdrewAll, uint withdrawnAmount) {
+    uint minAmountOut = amount * (PRICE_IMPACT_PRECISION - PRICE_IMPACT) / PRICE_IMPACT_PRECISION;
+
+    (uint lpAmount, uint reward) = POOL_DEPOSITOR.getDepositAmountOut(address(lpToken), amount);
+    lpAmount = lpAmount + reward;
```
Comment (belbix) [inline contracts/strategies/wombex/WombexStrategyBase.sol]: +=

File: contracts/strategies/UniversalLendStrategy.sol (hunk 1 above)
Comment (AlehNat) [reply contracts/strategies/UniversalLendStrategy.sol]: You are correct. It shouldn't be changed. Reverted.

File: contracts/strategies/radiant/Radiant2StrategyBase.sol (hunk 2 above)
Comment (AlehNat) [reply contracts/strategies/radiant/Radiant2StrategyBase.sol]: done

File: contracts/strategies/wombex/WombexStrategyBase.sol (hunk 3 above)
Comment (AlehNat) [reply contracts/strategies/wombex/WombexStrategyBase.sol]: done

**Pull request merged**